transformations/cmd_line.go

```diff
@@ -40,12 +54,22 @@ func cmdLine(data string) (string, error) {
 			}
 			space = false
 
-			ret = append(ret, byte(a))
+			ret = append(ret, a)
 		default:
-			b := unicode.ToLower(a)
-			ret = append(ret, byte(b))
```

Comment:
Anyways while a unicode lowercase was attempted here, by casting to byte the rune is truncated so it would have been a buggy conversion anyways